app/routes/cdo/edit/change-status.js

```diff
@@ -4,8 +4,30 @@ const getUser = require('../../../auth/get-user')
 const ViewModel = require('../../../models/cdo/edit/change-status')
 const { getCdo } = require('../../../api/ddi-index-api/cdo')
 const { updateStatus } = require('../../../api/ddi-index-api/dog')
-const { validatePayload } = require('../../../schema/portal/edit/change-status')
+const { validatePayload, duplicateMicrochipSchema } = require('../../../schema/portal/edit/change-status')
 const { addBackNavigation, addBackNavigationForErrorCondition } = require('../../../lib/back-helpers')
+const { ApiConflictError } = require('../../../errors/api-conflict-error')
+
+const changeStatusPostFailAction = async (request, h, error) => {
```

Comment:
I didn't think there was a particular need to extract this out to a method (unless you needed it for unit testing)??

Reply:
it's being used in the POST handler - so was wanting to make sure it's not repeating